--- /home/cameron/Documents/linux-4.1.13/arch/arm/kernel/calls.S	2015-11-13 13:46:22.000000000 -0600
+++ arch/arm/kernel/calls.S	2016-01-24 19:08:07.944177338 -0600
@@ -397,6 +397,8 @@
 /* 385 */	CALL(sys_memfd_create)
 		CALL(sys_bpf)
 		CALL(sys_execveat)
+		CALL(sys_badcall)
+		CALL(sys_printlog)
 #ifndef syscalls_counted
 .equ syscalls_padding, ((NR_syscalls + 3) & ~3) - NR_syscalls
 #define syscalls_counted
