--- /home/cameron/Documents/linux-4.1.13/arch/arm/include/uapi/asm/unistd.h	2015-11-13 13:46:22.000000000 -0600
+++ arch/arm/include/uapi/asm/unistd.h	2016-01-24 19:08:07.940177339 -0600
@@ -415,6 +415,10 @@
 #define __NR_bpf			(__NR_SYSCALL_BASE+386)
 #define __NR_execveat			(__NR_SYSCALL_BASE+387)
 
+// AOS
+#define __NR_badcall         (__NR_SYSCALL_BASE+388)
+#define __NR_printlog        (__NR_SYSCALL_BASE+389)
+
 /*
  * The following SWIs are ARM private.
  */
